apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/ListOrders/mapOrders.ts

```diff
@@ -23,7 +24,49 @@ export type OrderProps = {
   txLink: string
 }
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-export const mapOrders = (orders: Array<SwapOpenOrder>): Array<OrderProps> => {
-  return getMockOrders()
+export const mapOrders = (
+  orders: Array<SwapOrder>,
+  tokenInfos: Balance.TokenInfo[],
+  numberLocale: NumberLocale,
+  transactionInfos: TransactionInfo[],
+) => {
+  return orders.map((order) => {
+    const fromTokenInfo = tokenInfos.find((tokenInfo) => tokenInfo.id === order.from.tokenId)
+    const toTokenInfo = tokenInfos.find((tokenInfo) => tokenInfo.id === order.to.tokenId)
+    const id = `${order.from.tokenId}-${order.to.tokenId}-${order.utxo}`
+    const fromLabel = fromTokenInfo?.ticker ?? fromTokenInfo?.name ?? '-'
+    const toLabel = toTokenInfo?.ticker ?? toTokenInfo?.name ?? '-'
+    const tokenAmount = BigNumber(Quantities.denominated(order.to.quantity, toTokenInfo?.decimals ?? 0)).toFormat(
+      numberLocale,
+    )
+    const tokenPrice = BigNumber(
+      Quantities.quotient(
+        Quantities.denominated(order.from.quantity, fromTokenInfo?.decimals ?? 0),
+        Quantities.denominated(order.to.quantity, toTokenInfo?.decimals ?? 0),
+      ),
+    ).toFormat(numberLocale)
+    const txId = order.utxo.split('#')[0]
+    const total = BigNumber(Quantities.denominated(order.from.quantity, fromTokenInfo?.decimals ?? 0)).toFormat(
+      numberLocale,
+    )
+    const matchingTxInfo = transactionInfos.find((tx) => tx.id === txId)
+    const submittedAt = matchingTxInfo?.submittedAt
+    const txLink = `https://cardanoscan.io/transaction/${txId}`
+    const date = isString(submittedAt) ? new Date(submittedAt).toISOString() : ''
+    return {
+      tokenPrice,
+      tokenAmount,
+      id,
+      assetFromLabel: fromLabel,
+      assetToLabel: toLabel,
+      date,
+      txId,
+      total,
+      txLink,
+      fromTokenInfo,
+      toTokenInfo,
+      provider: order.provider,
+      poolUrl: `https://google.com`, // TODO: get pool url from order.provider
```

Comment:
the list of the providers is `short`, we can create a function that returns it based on a constant and fallback to the muesli in case there is a new one, until we add it.